db-operator/crds/readme.md

```diff
@@ -0,0 +1,3 @@
+# Files in this folder should get generated and not manual edited.
```

Comment:
readme needs an update to point to the db-operator git project and then copy files to this place.